frontend/assets/images/inline-svgs/raw/bundle-adfree.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="28px" height="30px" viewBox="0 0 28 30" enable-background="new 0 0 28 30" xml:space="preserve">
+<g>
+	<path fill="#333333" d="M8,10c5.523,0,10-4.477,10-10c0,5.523,4.477,10,10,10c-5.523,0-10,4.478-10,10C18,14.478,13.523,10,8,10"/>

```

Comment:
_Maybe_ these `fill` values can be moved to the CSS to make these SVGs more reusable?